### ISSUE TITLE
Navigating with the mouse in browsers chromium

### ISSUE THREAD

**Issue by Nikita34196** [nvaccess/nvda#9515]
Hello. Follow these steps.
Open any site. Preferably an article. Try to hover the mouse on the text and the link using the mouse.
Nvda should read the text, but it does not. Reads the first two words.
Browsers. edge 75.0.138.0 google chrome 75.0.3766.2
nvda alpha-17077,7f274eca

**Comment by DrSooom** [comment]
@gdata1: Please change the "Text unit resolution" value in the Mouse NVDA Settings and report their results. Thanks.

**Comment by gdata1** [comment]
I did, but the result is the same

**Comment by gdata1** [comment]
In browsers like the old Microsoft edge Mozila I do not see the problem.

**Comment by gdata1** [comment]
@DrSooom I have tried to change the mouse settings in Nvda Nothing has changed the problem there.

**Comment by Qchristensen** [comment]
I think this is related to a general issue of NVDA and the mouse in Chrome / Chromium browsers.  I have noticed that hovering the mouse over elements sometimes will read them, and other times NVDA reports nothing.  Visually the browser recognises that the mouse is hovering over a particular element - for instance usernames of dates in comments on this issue it will highlight them and / or pop up a contact card for the user or tooltip showing the exact date and time a comment was posted, etc.

I also had a user report (in the NVDA Screen Reader group on Facebook) that they cannot use the "Move navigator object to mouse" in Chrome, which I can also replicate with NVDA 2019.2.1 and Chrome 77.0.3865.90.  I tried several earlier versions of NVDA, as I was sure this used to work, but back to version 2018.3.2 it doesn't work, which makes me inclined to think it's a change in Chrome rather than NVDA.

For instance, in Firefox, in desktop keyboard layout, you can hover the mouse over a section of the page, press NVDA+NUMPAD MULTIPLY to move navigator object to mouse, then NUMPAD PLUS to read from the current review cursor location, and it will read from where the mouse was.  This doesn't work at all in Chrome (or I can't find an example where it does - in fact after pressing NVDA+NUMPAD MULTIPLY, pressing NUMPAD PLUS reads nothing, not even from the top of the page.

**Comment by leonardder** [comment]
There are some bugs in the IA2 implementation of Chrome that cause this. For some elements on the page, the location data by IA2 is just not correct.

May be things improve if Microsoft invests in native UIA in Edge Chromium

**Comment by Adriani90** [comment]
@gdata1 how is this behavior in Edge Cromium?

**Comment by gdata1** [comment]
@Adriani90 this issue has not yet been fixed. especially on some sites. for example, www.vk.com

**Comment by gdata1** [comment]
I checked for edge 81 stable version and edge 84 Canary

**Comment by Adriani90** [comment]
cc: @aleventhal, @ObjectInSpace were there any plans to improve IAccessible2 in Chrome regarding offsets and mouse tracking? This issue is for a long time now and alot of users ask for improvement.

**Comment by Adriani90** [comment]
Closing as duplicate of the newer #11845 which contains more detailsed discussion and referenced issues.
